api/v1alpha1/ratelimit_types.go

```diff
@@ -138,6 +138,13 @@ type SourceMatch struct {
 	// +kubebuilder:validation:MinLength=1
 	// +kubebuilder:validation:MaxLength=256
 	Value string `json:"value"`
+
+	// Invert specifies whether the value match result will be inverted.
+	//
+	// +optional
+	// +kubebuilder:default=false
+	// +notImplementedHide
+	Invert bool `json:"invertMatch,omitempty"`
```

Comment:
can this be made into a `*bool` ? to ensure consistency across all optional fields in the API